spec/rspec/support/output_formatter_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+require 'rspec/support/output_formatter'
+require 'rspec/matchers/fail_matchers'
+
+module RSpec
+  module Support
+    describe OutputFormatter, ".format" do
```

Comment:
> You're totally right!

It's always a good idea to see each spec go red at least once by writing it before the implementation or, if you write it afterwards, "breaking" the implementation temporarily.  It guards against this kind of issue by ensuring your tests are actually testing what you intend them to.